rundoctests.py

```diff
@@ -9,11 +9,52 @@
 import os
 import sys
 import doctest
-import resource
+from doctest import DocTestParser, Example, SKIP
+from multiprocessing import Process
+if os.name != 'nt':
+    import resource
```

Comment:
Move this import inside the block below where you actually use it.

Reply:
I don't like that, but if you insist.

Reply:
I take it back--in this case it's only used at module-level anyways so I agree this should be merged into that block.